packages/client/src/app/components/modals/crafting/Recipes/RecipeCard.tsx

```
@@ -70,7 +100,26 @@ export const RecipeCard = ({
+              subTitleText='Grants'
+              subTitleContent={
+                <>
+                  {recipe.experience} <img style={{ width: '1.2vw' }} src={ExpIcon} />
+                </>
+              }
+              description={''}
+              leftSideText='Requirements'
+              leftSideContent={getRequirements()}
+              rightSideText='Costs'
+              rightSideContent={getCosts()}
+            />,
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)